src/handlers/autolabel.rs

```diff
@@ -6,6 +7,15 @@ use crate::{
 use anyhow::Context as _;
 use tracing as log;
 
+/// Key for the state in the database
+const AUTOLABEL_KEY: &str = "autolabel";
+
+/// State stored in the database
+#[derive(Debug, Default, serde::Deserialize, serde::Serialize)]
+struct AutolabelState {
+    new_pr_labels_applied: bool,
+}
```

Comment:
```suggestion
#[derive(Debug, Default, serde::Deserialize, serde::Serialize)]
struct AutolabelState {
    has_new_labels: bool,
}
```

?

Reply:
That sounds a bit confusing/not fully accurate to me, because these labels might not necessarily exist on the PR *right now*, we just remember that we have already tried to apply the `new_pr` labels already. I added a comment to explain this, let me know what you think.

Reply:
> we just remember that we have already tried to apply the new_pr labels already

I see. So in plain English it would be:
> This flag registers if a PR was previously labeled with the predefined set of labels we apply to all new PRs"

Is that correct? Then maybe `already_autolabeled` (or similar)? But I agree, a bit hard to summarize in a short variable name.

(btw, this flag purpose would be more clearer if named `is_new_pr` or similar)

Reply:
- `new_pr_already_autolabeled`
- `is_new_pr_labels_applied`

Do you like any of these? :laughing: I think that the comment is enough, we probably can't find a completely perfect name for this flag.

Reply:
agreed. let's drop the subject. sorry for wasting some time on this. I hope the comment is enough to a casual reader.